gfx/video_driver.c

```diff
@@ -2464,7 +2464,7 @@ void video_driver_frame(const void *data, unsigned width,
          if (video_info.fps_show)
          {
             snprintf(video_info.fps_text, sizeof(video_info.fps_text),
-                  "||  FPS: %6.1f ", last_fps);
+                  "|| FPS: %6.1f ", last_fps);
```

Comment:
That space was there on purpose so it looked uniform

Reply:
Yeah, please change that part back.